### PR TITLE
Added djmoney.money.Money.__round__

### PR DESCRIPTION
**Pull request by ZuluPro**
There's a failing test but not due to my patch.

### CODE REVIEW

**Comment by Stranger6667** [comment]
Hi @ZuluPro !
On the first glance, it seems like a duplicate of #391. I'll take a look today

**Comment by Stranger6667** [comment]
And thank you for the interest :)

**Comment by Stranger6667** [comment]
Aha, it returns a numeric value, instead of `Money` instance. I was thinking about rounding `Money` instances, which seems more logical to me - all math operations should return instances of the same type.

File: djmoney/money.py [hunk 1]
```diff
@@ -29,6 +29,9 @@ def __float__(self):
                       "'amount' attribute instead.", DeprecationWarning)
         return float(self.amount)
 
+    def __round__(self, ndigits):
+        return self.amount.__round__(ndigits)
```
Comment (benjaoming) [inline djmoney/money.py]: This returns a different type, so will prefer the implementation from #391

**Comment by benjaoming** [comment]
Thanks @ZuluPro - hoping that it's cool we just close this one and fast-track #391 for release... will get it out right meow! :smile_cat:

**Comment by ZuluPro** [comment]
NP men !

**Pull request closed without merging**